app/controllers/candidates/placement_requests/cancellations_controller.rb

```diff
@@ -23,7 +23,8 @@ def create
           notify_candidate @cancellation
           @cancellation.sent!
 
-          Bookings::Gitis::SchoolExperience.from_cancellation(@cancellation, :cancelled_by_candidate)
+          status = placement_request.booking.present? ? :cancelled_by_candidate : :withdrawn
```

Comment:
It looks like you can cancel a booking as a school, but that would register as cancelled by the client here?

Reply:
Just double checked, cancelling as a school goes through the controllers/schools/confirmed_bookings/cancellations/notification_deliveries_controller.rb. This controller is candidate cancellation only.

Reply:
Oh that's confusing 😕  but fair enough I guess its consistent